scripts/core/superdesk-api.d.ts

```diff
@@ -5,18 +5,18 @@ declare module 'superdesk-api' {
 
     type OrderedMap<K, V> = import('immutable').OrderedMap<K, V>;
 
```

Comment:
looks like there are no changes here apart from whitespace, can you revert it then?